CHANGELOG.md

```diff
@@ -1,5 +1,110 @@
 # Changelog
 
+## v6.0.0-beta.8 - 2017-10-05
+
+### Breaking changes
+
+* Changed `New-Service` to return error when given unsupported `-StartupType` and fixed `Set-Service` icon failing test. (#4802)
+* Allow `*` to be used in registry path for `Remove-Item`. (#4866)
+* Remove unsupported `-ShowWindow` switch from `Get-Help`. (#4903)
+* Fix incorrect position of a parameter which resulted in the args passed as input instead of as args for `InvokeScript()`. (#4963)
+
+### Engine updates and fixes
+
+* Make calls to `void CodeMethod` work. (#4850) (Thanks @powercode)
+* Get `PSVersion` and `GitCommitId` from the `ProductVersion` attribute of assembly (#4863) (Thanks @iSazonov)
+* Fix `powershell -version` and built-in help for `powershell.exe` to align with other native tools. (#4958 & #4931) (Thanks @iSazonov)
+* Load assemblies with `Assembly.LoadFrom` before `Assembly.Load` when the file path is given. (#4196)
+* Add a generic file watcher function in `HelpersCommon.psm1`. (#4775)
+* Update old links and fix broken links in `docs/host-powershell/README.md`. (#4877)
+* Fix when importing remote modules using version filters (and added tests). (#4900)
+* Enable transcription of native commands on non-Windows platforms. (#4871)
+* Add a new line to `CommandNotFoundException` error string. (#4934 & #4991)
+* Fix bug where PowerShell would exit with an error within an SSH remoting connection on Linux. (#4993)
+* Fix issues with expression redirected to file. (#4847)
+
+### General cmdlet updates and fixes
+
+* Added `Remove-Service` to Management module. (#4858) (Thanks @joandrsn)
+* Added functionality to set credentials on `Set-Service` command. (#4844) (Thanks @joandrsn)
+* Fix `Select-String` to exclude directories (as opposed to individual files) discovered from `-Path`. (#4829) (Thanks @iSazonov)
+* `Get-Date` now supports more argument completion scenarios by adding `ArgumentCompletionsAttribute`. (#4835) (Thanks @iSazonov)
+* Exclude `-ComObject` parameter of `New-Object` on unsupported (currently non-Windows) platforms. (#4922) (Thanks @iSazonov)
+* Updated default `ModuleVersion` in `New-ModuleManifest` to `0.0.1` to align with SemVer. (#4842) (Thanks @LDSpits)
+* Add Multipart support to web cmdlets. (#4782) (Thanks @markekraus)
+* Add `-ResponseHeadersVariable` to `Invoke-RestMethod` to enable the capture of response headers. (#4888) (Thanks @markekraus) 
+* Initialize web cmdlets headers dictionary only once. (#4853) (Thanks @markekraus)
+* Change web cmdlets `UserAgent` from `WindowsPowerShell` to `PowerShell`. (#4914)
```

Comment:
Not that I really care, but I'm not credited here for #4914

Reply:
I think you should get credited, but for me this raises a more important issue we should address post beta.8 which is it seems the changelog draft should be autogenerated so that all the issue numbers and credit is pulled from GitHub.  Script can scrub the MSFT contributions.  Organizing and rewording will still be manual.

Reply:
Adding the credit while I make some other fixes to spelling and lint errors